dlk/python/dlk/templates/src/func/impl/arm_neon/quantized_conv2d_tiling.cpp

```diff
@@ -302,10 +302,10 @@ void QuantizedConv2DTiling(QUANTIZED_NOT_PACKED input[],
                   for (unsigned int kc = 0; kc < kw; ++kc) {
                     uint32x4_t nk = vld1q_u32(&notk[kr][kc][0]);
                     uint8x16_t nk8 = vreinterpretq_u8_u32(nk);
-                    uint32x4_t in = vdupq_n_u32(in_tile[row + kr][col + kc][0]);
+                    uint32x4_t in = vdupq_n_u32(static_cast<QUANTIZED_PACKED::T>(in_tile[row + kr][col + kc][0]));
```

Comment:
This cast all the way is a bit heavy to comprehend actually.
Wonder could we just add something like
```cpp
explicit T Raw() const { return val; }
```
Into the class.

And write such code as
```
uint32x4_t in = vdupq_n_u32(in_tile[row + kr][col + kc][0].Raw());
```
instead?

As it seems there will be a lot of it. :thinking:

Reply:
Implemented `Raw`.